packages/manager/src/views/SPAPropertyDetailPage/SPAPropertyDetailPage.tsx

```diff
@@ -84,7 +91,20 @@ export const SPAPropertyDetailPage = (): JSX.Element => {
   );
 
   const lineChartLegend = Object.keys(monthlyDeployChart?.data || {}).map((key) => ({ name: key }));
+  const Totaldeployment = deploymentCount.data?.reduce((acc, obj) => acc + obj.count, 0);
 
+  const averageDeploymentTime = [
+    useGetMonthlyDeploymentsTime(propertyIdentifier, spaProperty).data,
+    useGetQuarterlyDeploymentsTime(propertyIdentifier, spaProperty).data,
+    useGetHalfYearlyDeploymentsTime(propertyIdentifier, spaProperty).data,
+    useGetYearlyDeploymentsTime(propertyIdentifier, spaProperty).data
+  ];
+  console.log('>>>', useGetMonthlyDeploymentsTime(propertyIdentifier, spaProperty).data);
```

Comment:
Kindly remove the console log and the hook inside it

Reply:
done